Packs/Okta/Integrations/Okta_v2/Okta_v2.yml

```diff
@@ -1449,6 +1449,20 @@ script:
       predefined:
       - 'true'
       - 'false'
+    - auto: PREDEFINED
+      defaultValue: 'false'
+      description: Revokes the User's existing sessions if true.
```

Comment:
Since the new endpoint revokes and send a temporary password let add it to the description: Revokes the User's existing sessions and return a temporary password if true

Reply:
Done